src/components/CodeEditor/CodeEditor.jsx

```diff
@@ -105,7 +161,8 @@ export default function CodeEditor(props) {
             sx={{ margin: "0 0.5rem" }}
             color="secondary"
             variant="contained"
-            disabled
+            disabled={deployment === null} 
```

Comment:
we might just wanna get rid of the disabled attribute all together. Users should be able to deploy if they want to

Reply:
nvm i think i misunderstood this button lol